### ISSUE TITLE
Bug(prefer-meta-last): false positive when inside object

### ISSUE THREAD

**Issue by MartinCura** [marcalexiei/eslint-plugin-zod-x#70]
### 🧩 Description

I didn't expect this to be invalid.

```ts
export const baseEventPayloadSchema = z.object({
  type: z.string(),
  action: z.string().meta({ description: "e.g. `payment.created`" }),
  //                ^^^^^ The `.meta()` methods should be the last one called

}
```

For now moved to its own schema even if it not used elsewhere:
```ts
const actionSchema = z.string().meta({ description: "e.g. `payment.created`" })

export const baseEventPayloadSchema = z.object({
  type: z.string(),
  action: actionSchema,
}
```

### 🪜 How to reproduce

Use first schema above.

### ✅ Expected behavior

I expected no rule violation, unless this is me not understanding the new `.meta()`. This line had `.describe()` before.

### 💻 System info

```shell
System:
    OS: Linux 6.16 Fedora Linux 42 (Workstation Edition)
    CPU: (22) x64 Intel(R) Core(TM) Ultra 9 185H
    Memory: 37.29 GB / 62.23 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh
  Binaries:
    Node: 22.16.0 - /home/martin/.nvm/versions/node/v22.16.0/bin/node
    npm: 10.9.2 - /home/martin/.nvm/versions/node/v22.16.0/bin/npm
    pnpm: 10.17.1 - /home/martin/.nvm/versions/node/v22.16.0/bin/pnpm
```
